bsd_license.txt

```diff
@@ -0,0 +1,27 @@
+For copyright information see the COPYRIGHT file included in the top-level
```

Comment:
If you're going to do this, is it worth changing the two references to the "above copyright notice" below to read something like "the above-mentioned copyright notice", since there is no longer a copyright notice above? Just a thought...